src/Core/src/Hosting/MauiHostEnvironment.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Maui.ApplicationModel;
+using Microsoft.Maui.Storage;
+
+namespace Microsoft.Maui.Hosting
+{
+	public class MauiHostEnvironment : IHostEnvironment
+	{
+		public string EnvironmentName
+		{ 
+			get
+			{
+#if DEBUG
+				return "Development";
+#else
+				return "Production";
+#endif
+			}
+
+			set => throw new System.NotImplementedException();
+		}
+
+		public string ApplicationName
+		{
+			get => AppInfo.Current.Name;
+			set => throw new System.NotImplementedException();
+		}
+
+		public string ContentRootPath
+		{ 
+			get => FileSystem.Current.AppDataDirectory;
```

Comment:
Does this need to be reconciled at all with BlazorHybridWebView?  Is it implicated with any of their content file processing/rooting?

Reply:
I don't think so. I'm not sure it even does anything right now. Removed it and make it throw `NotImplementedException` so that there are no expectations on this doing anything for MAUI apps until we can actually make it useful.